### ISSUE TITLE
Resources

### ISSUE THREAD

**Issue by vickimcfadden** [GSA/oasis-plus#63]
- As a buyer, I need access to tools and resources, so that I can receive training, gain access to pricing tools, conduct procurements and manage my task orders, or read more on the OASIS+ contract.
- As a seller, I need access to tools and resources, so that I can receive training, manage my contacts, or learn more about other sellers / find opportunities for sub-contracting. 

- [Content](https://docs.google.com/document/d/1uaFZULg_IqJplogKvAXYwSpD9ST4hrBSG-21B8qnV_c/edit)
- [Design](https://www.figma.com/file/lWfZDygV7WbvGlea2PIrpa/OASIS-Playground?type=design&node-id=2692-10386&mode=design&t=8NNibiYPyFM551vP-4)

**Status**

- [x]  Content created
- [x]  Content reviewed by SD
- [x]  Content reviewed by SME
- [x]  Design finalized
- [x]  Content finalized
- [x]  Page developed

**Comment by mark-meyer** [comment]
@klohman This is ready for you to look at:

https://federalist-76c8d2ee-2655-47f7-8fa4-63811842d0a5.sites.pages.cloud.gov/preview/gsa/oasis-plus/mmeyer/resources/resources/

One note: USWDS tables by default have fluid column widths based on content. It looked a little strange to have some tables with different columns widths, so I pinned the first column to 33%. It seems reasonable to me on mobile and desktop, but it's easy to change if you want to see something else.

**Comment by klohman** [comment]
@mark-meyer Looks good - table widths are working well. 

One question (and not necessarily related to this story - the url path for the training link (at bottom) should be  /events-training. That page hasn't been developed yet, so not sure if that should be updated now or when we pull in that story. 

Other than that - ready for QA.

**Comment by vickimcfadden** [comment]
Created a new card for updating Training links, [here](https://github.com/orgs/GSA/projects/111/views/1?pane=issue&itemId=54479803)

**Comment by vickimcfadden** [comment]
SD QA review complete. @ekidenda @PowderD ready for your review.

**Comment by mark-meyer** [comment]
This now in the main branch, so it should be available here: https://federalist-76c8d2ee-2655-47f7-8fa4-63811842d0a5.sites.pages.cloud.gov/site/gsa/oasis-plus/resources/

**Comment by ekidenda** [comment]
The CALC Demo video goes to 404 - and is not available. A new demo video is planned and will be ready for the MVP phase 2 timeframe. I added a comment on the content doc about this.

**Comment by rosamundtgov** [comment]
@ekidenda @PowderD ready for your review.

**Comment by ekidenda** [comment]
The page looks good. 

Pending some training video links to be added prior to launch. In the meantime...

- [Symphony Procurement Suite](https://oasis.app.cloud.gov/) @PowderD is this single entry point the one to use here?

Also, I see a placeholder for one registration training video, but noticed that Symphony has two different registration videos for Buyers and Sellers:
@PowderD do we need to link both registration videos below to the page?
- Sellers: [https://gettingstarted.apexlogic.com/support/solutions/articles/35000213221-how-to-register-as-a-business-user](url) and
- Buyers: [https://gettingstarted.apexlogic.com/support/solutions/articles/35000211063-how-to-complete-government-registration]...?

**Comment by PowderD** [comment]
@ekidenda Yes, we're putting in a placeholder for the training videos until the OASIS+ environment is built out.  That will be the single point of entry URL.  There isn't a plan to change the URL at this point.   I'll keep you up to date if that changes.
